play-json/js/src/test/scala-3/play/api/libs/json/EnumSpec.scala

```diff
@@ -11,21 +11,14 @@ final class EnumSpec extends AnyWordSpec with Matchers {
   "EnumFormat" should {
     import TestEnums.EnumWithDefaultNames._
 
-    // https://gitter.im/lampepfl/dotty?at=5ee22d1e7b6da9126a8b4a51 ¯\_(ツ)_/¯
```

Comment:
I can't access that chat log anymore but it it looks like the problem is now fixed :wink: